src/guessTheNumber.js

```diff
@@ -0,0 +1,16 @@
+const randomNum = Math.floor(Math.random() * 100)
+export const guessTheNumber = (input) => {
+  if(Number.isInteger(input) && input < randomNum) {
+    return 'Your guess is kinda low'
+  } else if(Number.isInteger(input) && input > randomNum) {
```

Comment:
It would be much easier if the else if statements just started in the another line after the closing bracket. This applies to all.